### PR TITLE
Prepare serializing of ABAP Language Version

### PR DESCRIPTION
**Pull request by mbtools**
Final prep step: Pass ABAP Language Version of repo to serializer classes

### CODE REVIEW

**Review by larshp — APPROVED** [review]

**Comment by larshp** [comment]
oh, hit some of the target rules, https://github.com/abapGit/abapGit/pull/6525/checks?check_run_id=17338705220 suggest fixing these 4 :)

**Comment by larshp** [comment]
👍

File: src/objects/core/zcl_abapgit_serialize.clas.abap [hunk 1]
```diff
@@ -504,11 +508,11 @@ CLASS zcl_abapgit_serialize IMPLEMENTATION.
     IF mi_log IS BOUND.
       LOOP AT lt_unsupported_count ASSIGNING <ls_unsupported_count>.
         IF <ls_unsupported_count>-count = 1.
-          mi_log->add_error( iv_msg  = |Object type { <ls_unsupported_count>-obj_type } not supported, {
-                                       <ls_unsupported_count>-obj_name } ignored| ).
+          mi_log->add_error( |Object type { <ls_unsupported_count>-obj_type } not supported, | &&
+                             |<ls_unsupported_count>-obj_name } ignored| ).
```
Comment (CRCOOP) [inline src/objects/core/zcl_abapgit_serialize.clas.abap:512]: Hi, seems like this line: |<ls_unsupported_count>-obj_name } ignored| ). is missing a starting {.
It should be: |{ <ls_unsupported_count>-obj_name } ignored| ).

**Comment by mbtools** [comment]
thanks @CRCOOP... I should claim that I did that on purpose so @larshp can improve abaplint 😉

File: src/objects/core/zcl_abapgit_serialize.clas.abap [hunk 2]
```diff
@@ -504,11 +508,11 @@ CLASS zcl_abapgit_serialize IMPLEMENTATION.
...
         ELSE.
-          mi_log->add_error( iv_msg  = |Object type { <ls_unsupported_count>-obj_type } not supported, {
-                                       <ls_unsupported_count>-count } objects ignored| ).
+          mi_log->add_error( |Object type { <ls_unsupported_count>-obj_type } not supported, | &&
+                             |<ls_unsupported_count>-count } objects ignored| ).
```
[the remark this answers is not in the document]
Comment (mbtools) [reply src/objects/core/zcl_abapgit_serialize.clas.abap:515]: https://github.com/abapGit/abapGit/pull/6528

File: src/objects/core/zcl_abapgit_serialize.clas.abap (hunk 1 above)
Comment (mbtools) [reply src/objects/core/zcl_abapgit_serialize.clas.abap:512]: https://github.com/abapGit/abapGit/pull/6528

**Comment by larshp** [comment]
always another bug, but now there is also one more syntax error detected

**Pull request merged**